Src/AutoNSubstituteUnitTest/SubstituteRequestTest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Xunit;
+
+namespace Ploeh.AutoFixture.AutoNSubstitute.UnitTest
+{
+    public class SubstituteRequestTest
+    {
+        [Fact]
+        public void TargetTypeReturnsValueSpecifiedInConstructor()
+        {
+            // Fixture setup
+            var expectedType = typeof(int);
+            // Excercise system
+            var sut = new SubstituteRequest(expectedType);
+            // Verify outcome
+            Assert.Same(expectedType, sut.TargetType);

```

Comment:
Couldn't a Devil's Advocate pass the test by hard-coding `typeof(int)` in the implementation?